assignments/this.js

```diff
@@ -13,14 +13,60 @@
 
 // code example for Window Binding
 
+window._ = {};
+
 // Principle 2
 
 // code example for Implicit Binding
 
+function foo() {
```

Comment:
for implicit binding you are going to want to make an object and define a value then call that value some where else in the object. What you are binding the this keyword in this example is to the object it is currently inside of, ex. 
`const Talia = {
  name: 'Talia',
  logger: function() {
    console.log(this.name)
  }
}`

Reply:
This will log 'Talia' as the this is referring to the Talia object as a whole and the .name is referring to the key 'name'